packages/asset-swapper/src/utils/market_operation_utils/multihop_utils.ts

```diff
@@ -0,0 +1,36 @@
+import { BigNumber } from '@0x/utils';
+import * as _ from 'lodash';
+
+import { FeeSchedule, SourceInfo, TokenAdjacencyGraph } from './types';
+
+/**
+ * Given a token pair, returns the intermediate tokens to consider for two-hop routes.
+ */
+export function getIntermediateTokens(
+    makerToken: string,
+    takerToken: string,
+    tokenAdjacencyGraph: TokenAdjacencyGraph,
```

Comment:
What will this table look like? There are no X/ETH pairs?

Reply:
yeah, no X/ETH pairs because we pretty much always want to try ETH as the intermediate token. This is what I'm going with for now: 
```
const TOKEN_ADJACENCY_GRAPH = {
    // renBTC: wBTC
    '0xeb4c2781e4eba804ce9a9803c67d0893436bb27d': ['0x2260fac5e5542a773aa44fbcfedf7c193bc2c599'],
    // TUSD: USDC, DAI
    '0x0000000000085d4780B73119b644AE5ecd22b376': ['0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', '0x6b175474e89094c44da98b954eedeac495271d0f'],
    // USDT: USDC, DAI
    '0xdac17f958d2ee523a2206206994597c13d831ec7': ['0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', '0x6b175474e89094c44da98b954eedeac495271d0f'],
    // COMP: USDC
    '0xc00e94cb662c3520282e6f5717214004a7f26888': ['0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'],
    // LEND: USDC
    '0x80fB784B7eD66730e8b1DBd9820aFD29931aab03': ['0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'],
    // SNX: USDC
    '0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f': ['0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48'],
};
```